### PR TITLE
Added select/deselect all users in role screen

### PR DESCRIPTION
**Pull request by agackovka**
fixes https://github.com/OHDSI/WebAPI/issues/507

### CODE REVIEW

File: js/components/role-details.js [hunk 1]
```diff
@@ -51,6 +51,13 @@ define(['knockout', 'text!./role-details.html', 'appConfig', 'ohdsi.util', 'data
         self.canDelete = ko.pureComputed(function() { return self.isAuthenticated() && self.roleId() && authApi.isPermittedDeleteRole(self.roleId()); });
         self.canSave = ko.pureComputed(function() { return self.canEditRole() || self.canEditRoleUsers() || self.canEditRolePermissions(); });
 
+        self.areUsersSelected = ko.pureComputed(function() { return !!self.userItems().find(user => user.isRoleUser()); });
+        
+        self.selectAllUsers = () => self.userItems().forEach(user => user.isRoleUser(true));
+        self.deselectAllUsers = () => self.userItems().forEach(user => user.isRoleUser(false));
+        
+        self.checkboxTitle = '<span class="fa fa-check" data-bind="css: { selected: $component.areUsersSelected }, click: function(){$parent.areUsersSelected()?$parent.deselectAllUsers():$parent.selectAllUsers();}"></span> Included';
```
Comment (pavgra) [inline js/components/role-details.js]: Look like not the best approach: you place markup in JS & define quite a specific handler in databinding. I'd utilize datatables' ability to re-use thead defined in table's template

**Comment by pavgra** [comment]
@chrisknoll, any comments from your side?

**Comment by chrisknoll** [comment]
I don't see any problems. I like the enhancement to the datatables binding to support bindings in the header.

**Comment by pavgra** [comment]
@chrisknoll , yep, this was required for my nested table headers in CC

**Pull request merged**